js/main.js

```diff
@@ -0,0 +1,298 @@
+        2
+      );
+
+      this.timeoutId = setTimeout(() => {
+        this.runTimer();
+      }, 10);
+    }
+
+    // レベルチェンジボタンを押した時に確認する
+    changeLevel() {
+      confirm(
+        "ゲームのレベルを変更しますか？\n変更すると現在のゲームはリセットされます"
+      );
+      if (confirm) {
+        location.reload();
+      }
+    }
+
+    // 難易度によってコメントを分岐
+    partComment() {
+      switch (this.getLevel()) {
+        case 3:
+          this.easyLevelComment();
+          break;
+        case 4:
+          this.normalLevelComment();
+          break;
+        case 5:
+          this.hardLevelComment();
+          break;
+      }
+    }
+
+    // 難易度Easyの時のコメント
+    easyLevelComment() {
+      if (this.timer.textContent <= 6.5) {
+        this.comment.textContent = this.commentList[4];
+      } else if (this.timer.textContent <= 8) {
+        this.comment.textContent = this.commentList[3];
+      } else if (this.timer.textContent <= 10) {
+        this.comment.textContent = this.commentList[2];
+      } else if (this.timer.textContent <= 12) {
+        this.comment.textContent = this.commentList[1];
+      } else {
+        this.comment.textContent = this.commentList[0];
+      }
+    }
+
+    // 難易度Normalの時のコメント
+    normalLevelComment() {
+      if (this.timer.textContent <= 12) {
+        this.comment.textContent = this.commentList[4];
+      } else if (this.timer.textContent <= 14) {
+        this.comment.textContent = this.commentList[3];
+      } else if (this.timer.textContent <= 16) {
+        this.comment.textContent = this.commentList[2];
+      } else if (this.timer.textContent <= 18) {
+        this.comment.textContent = this.commentList[1];
+      } else {
+        this.comment.textContent = this.commentList[0];
+      }
+    }
+
+    // 難易度Hardの時のコメント
+    hardLevelComment() {
+      if (this.timer.textContent <= 22) {
+        this.comment.textContent = this.commentList[4];
+      } else if (this.timer.textContent <= 24) {
+        this.comment.textContent = this.commentList[3];
+      } else if (this.timer.textContent <= 26) {
+        this.comment.textContent = this.commentList[2];
+      } else if (this.timer.textContent <= 28) {
+        this.comment.textContent = this.commentList[1];
+      } else {
+        this.comment.textContent = this.commentList[0];
+      }
+    }
+
+    getLevel() {
+      return this.level;
+    }
+
+    getCurrentNum() {
+      return this.currentNum;
+    }
+
+    addCurrentNum() {
+      this.currentNum++;
+    }
+
+    getTimeoutId() {
+      return this.timeoutId;
+    }
+
+    getStart() {
+      return this.start;
+    }
+  }
+
+  const easy = document.getElementById("js-easy");
+  const normal = document.getElementById("js-normal");
+  const hard = document.getElementById("js-hard");
+  const mask = document.getElementById("js-mask");
+  const select = document.getElementById("js-select");
+  const levelText = document.getElementById("js-level-text");
```

Comment:
[IMO]
自分はですけど
```suggestion
  const easyElem = document.getElementById("js-easy");
  const normalElem = document.getElementById("js-normal");
  const hardElem = document.getElementById("js-hard");
  const maskElem = document.getElementById("js-mask");
  const selectElem = document.getElementById("js-select");
  const levelTextElem = document.getElementById("js-level-text");
```

よく変数に入っているのがなんなのかを分かりやすくします

Reply:
変数に要素が入っているのを名前で分かりやすくするんですね！
ありがとうございます！
早速使わせて頂きます！